reactivesocket-tck-drivers/run.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-java -cp build/libs/reactivesocket-tck-drivers-0.2.2-SNAPSHOT.jar io.reactivesocket.tckdrivers.main.Main "$@"
+java -cp build/libs/reactivesocket-tck-drivers-0.2.3-SNAPSHOT.jar io.reactivesocket.tckdrivers.main.Main "$@"
```

Comment:
Can you either make this find the right version automatically, or use something like this http://stackoverflow.com/a/21360609/1542667